pkg/sriov/sriov_test.go

```diff
@@ -31,6 +32,72 @@ func (l *FakeLink) Type() string {
 
 var _ = Describe("Sriov", func() {
 
+	Context("Checking ApplyVFConfig function", func() {
+		var (
+			netconf *types.NetConf
+		)
+
+		BeforeEach(func() {
+			netconf = &types.NetConf{
+				Master:      "i4",
+				DeviceID:    "0000:af:06.0",
```

Comment:
Add invalid tests case according to regex
such as 
1. invalid char
2. wrong lenght 
...

Reply:
added